src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FailOverSupportedConfigurationClient.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure;
+using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class FailOverSupportedConfigurationClient : IConfigurationClient
+    {
+        private const int HttpStatusRequestThrottled = 429;
+
+        private readonly IEnumerable<LocalConfigurationClient> clients;
+
+        private DateTimeOffset retryPrimaryConfigClientAfter;
```

Comment:
I don't understand why primary would be treated specially.

Reply:
Are you suggesting we always try primary first? That way we make our logic dumber because we already know primary store is down and we still try primary.
Or are you suggesting we store backoff time for all replicas as well?
We decided during the design discussion that we would not give any special treatments to the replicas and will always try the replicas in a sequential manner.

Reply:
> Or are you suggesting we store backoff time for all replicas as well?

Yeah, that's what I was suggesting. I don't remember mentioning in the meeting that we wouldn't have backoff for replicas. Was there reasoning to have it for primary and not replica?